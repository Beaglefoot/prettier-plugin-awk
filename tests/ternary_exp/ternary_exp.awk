{
  1==1?"true": 1 + 2
}
