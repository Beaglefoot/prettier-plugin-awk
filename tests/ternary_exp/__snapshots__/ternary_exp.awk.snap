{
  1==1?"true": 1 + 2
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{ 1 == 1 ? "true" : 1 + 2 }
